crates/matrix-sdk-base/src/store/integration_tests.rs

```diff
@@ -577,7 +577,7 @@ macro_rules! statestore_integration_tests {
                     let timeline = &sync.rooms.join[room_id].timeline;
                     let events: Vec<SyncRoomEvent> = timeline.events.iter().cloned().map(Into::into).collect();
 
-                    stored_events.append(&mut events.clone());
+                    stored_events.append(&mut events.iter().rev().cloned().collect());
```

Comment:
This looked weird before and looks even weirder now. If you change `append` to `extend` you can remove both the `&mut` and the `.collect()` while preserving behavior.